plugins/www/Optimove.js

```diff
@@ -6,3 +6,29 @@ var exec = require('cordova/exec');
 window.initialize = function (inAppConsentStrategy, success, error) {
     exec(success, error, 'OptimoveSDKPlugin', 'initialize', [inAppConsentStrategy]);
 };
+window.setUserId = function (userId, success, error) {
+    exec(success, error, 'OptimoveSDKPlugin', 'setUserId', [userId]);
+}
+window.setUserEmail = function (userEmail, success, error) {
+    exec(success, error, 'OptimoveSDKPlugin', 'setUserEmail', [userEmail]);
+};
+
+/* eventParams is nullable*/
```

Comment:
I think eventually we should try and either use TS for the plugin sources, or add a TS interfaces definition file to the project, so perhaps these annotations are redundant.

Alternately, we could use JSDoc to fully document the methods.

What do you think?

Reply:
OK after discussion, we will defer adding docblocks etc. until we look at using TS for the JS bridge layer.